src/Tribe/Utils/DST.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+class Tribe__Events__Utils__DST {
+
+	/**
+	 * @var bool|null
+	 */
+	protected $_in_dst;
+
+	/**
+	 * Tribe__Events__Utils__DST constructor.
+	 *
+	 * @param int|string $time    Either a UNIX timestamp or an english format date.
+	 * @param null|bool  $_in_dst An injectable DST status, meant for tests.
+	 */
+	public function __construct( $time, $_in_dst = null ) {
+		if ( ! is_numeric( $time ) ) {
+			$time = strtotime( $time );
+		}
+
+		$this->time    = $time;
+		$this->_in_dst = $_in_dst;
+	}
+
+	/**
+	 * Whether the current time is in DST or not.
+	 *
+	 * @return bool
+	 */
+	public function is_in_dst() {
+		return is_null( $this->_in_dst ) ? (bool) date( 'I', $this->time ) : $this->_in_dst;
```

Comment:
Re `date( 'I', $this->time )` - will the result always be what we expect if there is a mismatch between PHP's date.timezone setting and WP's `timezone_string` option?